resources/views/welcome.blade.php

```diff
@@ -11,7 +11,7 @@
 <div class="bg-gray-100 font-sans relative z-0">
     <div class="max-w-6xl mx-auto pt-8">
         <p class="mb-6 text-black-lighter">
-            Showing versions for {{ $projects->count() }} active projects and packages
+            Showing versions for <span id="projectCounter">{{ $projects->count() }}</span> active projects and packages
```

Comment:
Nitpicky but can you snake_case this id like you did with `id="project_{{ $project->id }}"` below please?

Reply:
Snake it is :)